code/modules/hydroponics/hydroponics.dm

```diff
@@ -215,25 +251,23 @@
 				adjustHealth(-rand(1,5) / rating)
 
 			// Harvest code
-			if(age > myseed.production && (age - lastproduce) > myseed.production && (!harvest && !dead))
-				nutrimentMutation()
 				if(myseed && myseed.yield != -1) // Unharvestable shouldn't be harvested
-					harvest = 1
+					harvest = TRUE

```

Comment:
is this indentation intentional? its now falling under the ``age > myseed.lifespan`` control